homeassistant/components/assist_pipeline/websocket_api.py

```diff
@@ -371,3 +380,76 @@ async def websocket_list_languages(
             else pipeline_languages
         },
     )
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "assist_pipeline/device/capture",
+        vol.Required("device_id"): str,
+        vol.Required("timeout"): vol.All(
+            # timeout > 0
+            vol.Coerce(float),
+            vol.Range(min=0, min_included=False),
+        ),
+    }
+)
+@websocket_api.async_response
+async def websocket_device_capture(
+    hass: HomeAssistant,
+    connection: websocket_api.connection.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Capture raw audio from a satellite device and forward to client."""
+    pipeline_data: PipelineData = hass.data[DOMAIN]
+    device_id = msg["device_id"]
+
+    # timeout <= MAX_CAPTURE_TIMEOUT
+    timeout_seconds = min(MAX_CAPTURE_TIMEOUT, msg["timeout"])
```

Comment:
Capture this in the validation schema by setting `max`

Reply:
Do you think it's sufficient to just document the max then? Otherwise, clients don't know the max until they call with too large a value.

Reply:
Yeah documentation is enough.